tests/ut/jit/test_decorator.py

```
@@ -1265,5 +1268,124 @@ def __init__(self, code: int, msg: str) -> None:
+    Before this fix, ``_compile`` only forwarded ``skip_ptoas`` and
+    ``platform`` — every other compile knob a user set on ``RunConfig``
+    (``strategy``, ``dump_passes``, ...) was silently dropped on the JIT path.
+    """
+
+    def test_run_config_compile_kwargs_maps_fields(self):
+        """Compile-side RunConfig fields map onto the ir.compile() parameter names."""
+        cfg = RunConfig(
+            strategy=OptimizationStrategy.DebugTileOptimization,
+            dump_passes=True,
+            compile_profiling=True,
+            save_kernels_dir="/tmp/jit_artifacts",
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/jit_artifacts"

(S108)